### PR TITLE
Added endpoints for Survey Module

### PR DESCRIPTION
**Pull request by marc-farre**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@digital-boss/n8n-nodes-humhub",
-    "version": "0.1.7",
+    "version": "0.1.8",
```
Comment (valentina98) [inline package.json]: Would you change the number back to version 0.1.7. We use the `npm version 0.1.8` command which runs the versioning scripts and adds an automatic commit.

File: nodes/version.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
 // Generated by genversion.
-export const version = '0.1.7';
+export const version = '0.1.8';
```
Comment (valentina98) [inline nodes/version.ts]: Would you change the number back to version 0.1.7. We use the `npm version 0.1.8` command which runs the version script and adds an automatic commit.

File: nodes/HumHub/HumHub.node.ts [hunk 3]
```diff
@@ -2145,7 +2169,158 @@ export class HumHub implements INodeType {
...
+                            Object.assign(qs, queryParameters);
+
+                            responseData = await humhubApiRequest.call(
+                                this,
+                                'GET',
+                                `/survey/${surveyId}/answers`,
+                                undefined,
+                                qs,
+                            );
+                        }
```
Comment (valentina98) [inline nodes/HumHub/HumHub.node.ts:2320]: There is a build error:

image

File: nodes/HumHub/descriptions/SurveyDescription.ts [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+	/* -------------------------------------------------------------------------- */
+
+	{
+		displayName: 'ID',
+		name: 'id',
+		type: 'number',
+		required: true,
+        typeOptions: {
+            numberStepSize: 1,
+        },
```
Comment (valentina98) [inline nodes/HumHub/descriptions/SurveyDescription.ts]: Please format the file

File: nodes/HumHub/descriptions/SurveyAnswerDescription.ts [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+	/* -------------------------------------------------------------------------- */
+	/*                                 surveyAnswer:get                                   */
+	/* -------------------------------------------------------------------------- */
+
+    {
+		displayName: 'ID',
+		name: 'id',
+		type: 'number',
+		required: true,
+        typeOptions: {
```
Comment (valentina98) [inline nodes/HumHub/descriptions/SurveyAnswerDescription.ts]: Please format the file

**Review by valentina98 — CHANGES_REQUESTED** [review]
Please fix and test all of the added endpoints

**Comment by marc-farre** [comment]
Thanks @valentina98 for your review. It should be correct now. Sorry, but I have no knowledge with npm, so I don't know how to build to test the endpoints...

**Review by samuk — APPROVED** [review]
@valentina98 would it be possible to review this please?

**Comment by marc-farre** [comment]
Thanks @valentina98 for merging this PR!
Could you create the new version with `npm version 0.1.8`?
Thanks

**Pull request merged**